src/modules/stripe/stripe-api.ts

```diff
@@ -13,7 +13,7 @@ import { createLogger, redactError } from "@/lib/logger";
 
 export const getStripeApiClient = (secretKey: string) => {
   const stripe = new Stripe(secretKey, {
-    apiVersion: "2022-11-15",
+    apiVersion: "2023-10-16",
```

Comment:
Can we update the apiVersion just like that? Shouldn't we also update webhooks?

Reply:
What do you mean by updating the webhooks? Do you have a link to the Stripe docs describing that?

Reply:
https://stripe.com/docs/upgrades

Are changes in payment intent relevant for us?
what about the error handling changes mentioned in the changelog?

Reply:
Yeah, I read that and I don't see anything relevant to us 👍